integration_tests/src/main/python/data_gen.py

```diff
@@ -756,14 +756,19 @@ def skip_if_not_utc():
 # Note: Current(2023/06/06) maxmium IT data size is 7282688 bytes, so LRU cache with maxsize 128
 # will lead to 7282688 * 128 = 932 MB additional memory usage in edge case, which is acceptable.
 @lru_cache(maxsize=128, typed=True)
-def gen_df_help(data_gen, length, seed):
-    rand = random.Random(seed)
+def gen_df_help(data_gen, length, seed_value):
+    rand = random.Random(seed_value)
     data_gen.start(rand)
     data = [data_gen.gen() for index in range(0, length)]
     return data
 
-def gen_df(spark, data_gen, length=2048, seed=0, num_slices=None):
+def gen_df(spark, data_gen, length=2048, seed=None, num_slices=None):
```

Comment:
Do we want a follow on issue to remove seed for gen_df and force us to go through the annotation?

Reply:
https://github.com/NVIDIA/spark-rapids/issues/9712